addons/mail/static/src/owl/components/activity/activity.scss

```diff
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
+}
+
+.o_Activity_detailsUserAvatar {
+    margin-right: 5px;
+    object-fit: cover;
+    height: 18px;
+    width: 18px;
+}
+
+.o_Activity_icon {
+    position: absolute;
+    top: auto;
+    left: auto;
+    bottom: -5px;
+    right: -5px;
+    width: 25px;
+    height: 25px;
+    object-fit: cover;
+    padding: 6px 5px;
+    text-align: center;
+    line-height: 1.2;
+    border-width: 2px;
+}
+
+.o_Activity_sidebar {
+    display: flex;
+    flex: 0 0 36px;
+    margin-right: 10px;
```

Comment:
spacer and probably margin-inline-end

Reply:
Are the activities print in "mirror" when text is rtl ?  If not, we still need margin-right, if yes, indeed we need margin-inline-end.

![image](https://user-images.githubusercontent.com/48204996/75666006-5bd77f80-5c75-11ea-85ab-fe4c7f64e340.png)

Reply:
You can go in the "languages" settings and set english to be rtl to test